buy_together_app/models.py

```diff
@@ -1,3 +1,51 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+
+class User(models.Model):
+    user_name = models.CharField(max_length=32, primary_key=True, null=False)
+    first_name = models.CharField(max_length=32)
+    last_name = models.CharField(max_length=32)
+    password = models.CharField(max_length=16)
+    access = models.BooleanField()
+
+    class Meta:
+        abstract = True
+
+
+class Supplier(User):
+    pass
+
+
+class Product(models.Model):
+    qr_code = models.CharField(max_length=32, primary_key=True)
+    product_name = models.CharField(max_length=32)
+    description = models.TextField()
+
```

Comment:
Products methods not here,if you can create a new branch based on upstream/main and create a seperate app for supplier_product and do the work there is better,because product is no longer in the buy_together_app